lib/src/base-styles/_nx-btn.scss

```diff
@@ -79,24 +79,28 @@
 }
 
 .nx-btn--error {
-  background-color: var(--nx-swatch-red-45);
-  border-color: var(--nx-swatch-red-45);
+  background-color: var(--nx-swatch-red-40);
+  border-color: var(--nx-swatch-red-40);
```

Comment:
If I'm not mistaken, if this was `transparent`, then `:hover` and `:active` wouldn't need to set `border-color`, which means only `:focus` is overridding it and then the `:hover` and `:focus` blocks could be re-ordered to remove the need for the `:hover:focus` block and thus remove the need for the `:hover:focus:active` block as well.

Reply:
> If I'm not mistaken, if this was `transparent`, then `:hover` and `:active` wouldn't need to set `border-color`, which means only `:focus` is overridding it and then the `:hover` and `:focus` blocks could be re-ordered to remove the need for the `:hover:focus` block and thus remove the need to specify `:hover:focus:active` rather than just `:active` as well.

Yep that was almost perfect. Just needed the border on `:active` to match Sketch https://github.com/sonatype/sonatype-react-shared-components/pull/481/commits/194217ed9bd1ffbdbc51b84adae392d681d99652